packages/jsii-pacmak/test/generated-code/__snapshots__/target-go.test.ts.snap

```diff
@@ -198,11 +191,44 @@ func StaticConsumer_Consume(args interface{}) {
 		[]interface{}{args},
 		false,
 		&returns,
-		map[reflect.Type]reflect.Type{},
 	)
 }
 
 
+`;
+
+exports[`Generated code for "@scope/jsii-calc-base": <outDir>/go/scopejsiicalcbase/scopejsiicalcbase.init.go 1`] = `
+package scopejsiicalcbase
+
+import (
+	"reflect"
+
+	_jsii_ "github.com/aws/jsii-runtime-go"
+)
+
+func init() {
+	_jsii_.RegisterClass(
+		"@scope/jsii-calc-base.Base",
```

Comment:
Is that the idiomatic way in Go to obtain a reflection type?

Reply:
Yes, it is. There is no way to get a type by name, because a type that isn't actually used (as in - being the type of some declared memory location) is not even present in the compiled program.